example/55-gardenlet.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: seedmanagement.gardener.cloud/v1alpha1
+kind: Gardenlet
+metadata:
+  name: <seed-name>
+  namespace: garden
+spec:
+  deployment: # gardenlet deployment parameters
+    helm:
+      ociRepository:
+        ref: local-skaffold/gardenlet/chart:v0.0.0
+    # image: # here you can overwrite the gardenlet image itself (typically not needed since the OCI URL above implicitly defines the image/version)
+    #   repository: europe-docker.pkg.dev/gardener-project/releases/gardener/gardenlet
+    #   tag: latest
+    #   pullPolicy: IfNotPresent
+    # imageVectorOverwrite: | # here you can overwrite the image repos + tags of the components deployed by gardenlet, e.g. etcd-druid, machine-controller-manager, kube-apiserver, etc.
+    #   Please find documentation in docs/deployment/image_vector.md
+    # componentImageVectorOverwrites: |
+    #   Please find documentation in docs/deployment/image_vector.md
```

```suggestion
  # image: # here you can overwrite the gardenlet image itself (typically not needed since the OCI URL above implicitly defines the image/version)
  #   repository: europe-docker.pkg.dev/gardener-project/releases/gardener/gardenlet
  #   tag: latest
  #   pullPolicy: IfNotPresent
  # imageVectorOverwrite: | # here you can overwrite the image repos + tags of the components deployed by gardenlet, e.g. etcd-druid, machine-controller-manager, kube-apiserver, etc.
  #   Please find documentation in docs/deployment/image_vector.md
  # componentImageVectorOverwrites: |
  #   Please find documentation in docs/deployment/image_vector.md
```

It's at the same level as `Helm`, isn't it?